tahowallet/extension
# Drop the `AddressOnNetwork` parameter for token import

Closes https://github.com/tahowallet/extension/issues/3401

## What
The token import should not be associated with any account. The balance should be checked for all accounts so the overview reflects the full balance with that token (though only on the asset's network).

## UI


https://github.com/tahowallet/extension/assets/23117945/2ed23346-7f79-4acf-aeb4-acad5dbbc3dd




## To Test
Accounts that have two of the same token(GGBoxs.com) are: 
- `0x208e94d5661a73360d9387d3ca169e5c130090cd`
- `0x63cf5f22c3719391e0014584bad67ecba6b9f98b`

Test token contract - `0xcf68f02d7dD6a4642AE6a77f6A3676D0CBC834c9`

To test the solution for the above data, you should first remove the token in order to add it again.

- [x] Try to import a custom token and check that the balances have been updated for all accounts.

## Testing Env

```
SUPPORT_UNVERIFIED_ASSET=true
```

Latest build: [extension-builds-3426](https://github.com/tahowallet/extension/suites/13403099239/artifacts/734005392) (as of Tue, 06 Jun 2023 10:47:15 GMT).
